src/CalcViewModel/Common/LocalizationSettings.h

```diff
@@ -265,34 +238,26 @@ namespace CalculatorApp
 
             std::wstring GetDecimalSeparatorStr() const
             {
-                std::wstring result;
-                result.push_back(m_decimalSeparator);
-                return result;
+                return std::wstring(1, m_decimalSeparator);
             }
 
             std::wstring GetNumberGroupingSeparatorStr() const
             {
-                std::wstring result;
-                result.push_back(m_numberGroupSeparator);
-                return result;
+                return std::wstring(1, m_numberGroupSeparator);
             }
 
             std::wstring GetNumberGroupingStr() const
             {
                 return m_numberGrouping;
             }
 
-            Platform::String ^ RemoveGroupSeparators(Platform::String ^ source) const
+            Platform::String^ RemoveGroupSeparators(Platform::String^ source) const
```

Comment:
Can you revert this line to stay consistent with the rest of the code?

Reply:
Done